tsdb/chunkenc/histo.go

```diff
@@ -258,9 +258,84 @@ func (a *HistoAppender) Appendable(h histogram.SparseHistogram) ([]Interjection,
 	if !ok {
 		return nil, nil, false
 	}
+	if counterReset(a.posbuckets, h.PositiveBuckets, a.posSpans, h.PositiveSpans) {
+		return nil, nil, false
```

Comment:
Counter resets need to be reflected in the doc comment above (as an additional case for a chunk not being appendable).